### PR TITLE
Add delete group features

### PR DESCRIPTION
**Pull request by scara82**
This Pr:
- Add delete & destroy methos in Group COntroller and routes;
- Add spec for happy path;
- Fixes the groupo model validation.

Trello Card: [https://trello.com/c/pNbZm2Z9](url)

### CODE REVIEW

**Comment by apalski** [comment]
LGTM!

File: app/controllers/groups_controller.rb [hunk 1]
```diff
@@ -22,6 +22,15 @@ def update
     respond_with @group, location: -> { groups_path }
   end
 
+  def delete
```
Comment (apalski) [inline app/controllers/groups_controller.rb]: Do you need delete and destroy?

Comment (elle) [reply app/controllers/groups_controller.rb]: Yep, just `destroy`

Comment (elle) [reply app/controllers/groups_controller.rb]: I think it is meant to be `show`

**Review by elle — APPROVED** [review]
Other than that on comment, LGTM 🚀

File: app/controllers/groups_controller.rb (hunk 1 above)
Comment (scara82) [reply app/controllers/groups_controller.rb]: Yep, thanks, it was useless. I deleted it!

**Comment by scara82** [comment]
@elle  and @apalski thanks for your comments! I'm about to merge it!

**Pull request merged**